.env.example

```diff
@@ -1,3 +0,0 @@
-# create local copy of this file and rename to .env
```

Comment:
could we leave the .env.example as is so future contributors will have the template? not sure if this is an accident, but if you do another commit adding this back everything should be fixed